auth-web/package-lock.json

```diff
@@ -36791,7 +36791,7 @@
       "dev": true
     },
     "fas-ui": {
-      "version": "git+ssh://git@github.com/bcgov/fas-ui.git#67e5772573db41e49c8d55b371aaeebba8622d11",
+      "version": "git+ssh://git@github.com/bcgov/fas-ui.git#55f72ab4c674875b081338d22bf26805eb262ed9",
```

Comment:
@seeker25 Should this hash be different?

Reply:
Probably this changed with package version update (npm i). Will wait for @seeker25 to reply.

Reply:
That hash is fine, 
https://github.com/bcgov/sbc-auth/pull/2199

I've updated the tag for FAS just in case.

Reply:
Oh, OK, Ketaki wasn't using the latest lock file.

This is fine.